net-misc/nextcloud-client/nextcloud_client-3.11.1.recipe

```diff
@@ -0,0 +1,163 @@
+	devel:libnextcloudsync$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	nextcloud_client$secondaryArchSuffix == $portVersion base
+	devel:libQt5Gui$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	extra_cmake_modules$secondaryArchSuffix
+	devel:libcmocka$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libkf5archive$secondaryArchSuffix
+	devel:libQt5Concurrent$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5DBus$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libqt5keychain$secondaryArchSuffix
+	devel:libqt5Network$secondaryArchSuffix
+	devel:libQT5Qml$secondaryArchSuffix
+	devel:libQt5QmlModels$secondaryArchSuffix
+	devel:libQt5Quick$secondaryArchSuffix
+	devel:libQt5QuickControls2$secondaryArchSuffix
+	devel:libQt5Sql$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libqt5Test$secondaryArchSuffix
+	devel:libQt5WebEngine$secondaryArchSuffix
+	devel:libQt5WebEngineWidgets$secondaryArchSuffix
+	devel:libQt5WebSockets$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libQt5Xml$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	#optional
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5WebChannel$secondaryArchSuffix
+	devel:libQt5Positioning$secondaryArchSuffix
+	devel:libQt5WebEngineCore$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:linguist$secondaryArchSuffix >= 5
+	cmd:rsvg_convert
+	"
+
+BUILD()
+{
+	cmake -Bbuild -S. $cmakeDirArgs \
+		-DCMAKE_INSTALL_BINDIR=$commandBinDir \
+		-DINCLUDE_INSTALL_DIR=$includeDir \
+		-DSYSCONF_INSTALL_DIR=$settingsDir \
+		-DCMAKE_BUILD_TYPE=Release -Wno-dev
+	make -Cbuild $jobArgs
+}
+
+INSTALL()
+{
+	make -Cbuild install
+
+	# don't mess with shared_mime_info files
+	rm -rf $dataDir/mime/{aliases,generic-icons,globs,globs2,icons,magic}
+	rm -rf $dataDir/mime/{mime.cache,subclasses,treemagic,types,version,XMLnamespaces}
+	# cleanup (not needed)
+	rm -rf $dataDir/{applications/icons}
+
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	local APP_NAME="Nextcloud Desktop Client"
+	local LONG_INFO="$SUMMARY"
+	local APP_SIGNATURE="application/x-vnd.nextcloud"
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		-e "s|@LONG_INFO@|$LONG_INFO|" \
+		-e "s|@APP_NAME@|$APP_NAME|" \
+		-e "s|@APP_SIGNATURE@|$APP_SIGNATURE|" \
+		$portDir/additional-files/nextcloud_client.rdef.in > nextcloud_client.rdef
+
+	addResourcesToBinaries nextcloud_client.rdef $binDir/nextcloud
+
+	addAppDeskbarSymlink $binDir/nextcloud Nextcloud
```

Comment:
`$binDir` needs to be replaced with $`commandBinDir` on these 2 lines.

Reply:
corrected 👍